.cspellignore

```diff
@@ -20,3 +20,4 @@ subcomponent
 subpage
 Traefik
 typesafe
+Undeprecate
```

Comment:
:thinking:

Reply:
I googled it and since NPM uses it it seems to be a correct word: https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions